templates/veoibd.yaml

```diff
@@ -41,18 +68,6 @@ Resources:
             Action:
               - 'sts:AssumeRole'
       Path: "/"
```

Comment:
you probably want to add `PublishToCloudwatchPolicy` to this role.

Reply:
How?